.travis.yml

```diff
@@ -10,33 +11,53 @@ env:
         # For now just hardcode the expected value.
         - CORE_COUNT=2
 
-    matrix:
-        - OS_TYPE=fedora
-          INSTALL_REQUIREMENTS="dnf repolist; dnf install -y gcc meson sudo langpacks-zh_CN ed ncurses vi"
+matrix:
+    include:
+        - os: linux
+          env:
+            - DISTRO_TYPE=fedora
+              INSTALL_REQUIREMENTS="dnf repolist; dnf install -y gcc meson sudo langpacks-zh_CN ed ncurses vi"
+          before_install:
+              - docker pull ${DISTRO_TYPE}
 
         # Note: Meson 0.44.0 is the newest version that works with Python
         # 3.4 available on OpenSuse Leap. See issue #430.
-        - OS_TYPE=opensuse
-          INSTALL_REQUIREMENTS="zypper refresh; zypper in -y gcc python3 python3-pip ninja sudo glibc-locale glibc-devel ed ncurses-utils vim; pip3 install meson==0.44.0"
+        - os: linux
+          env:
+            - DISTRO_TYPE=opensuse
+              INSTALL_REQUIREMENTS="zypper refresh; zypper in -y gcc python3 python3-pip ninja sudo glibc-locale glibc-devel ed ncurses-utils vim; pip3 install meson==0.44.0"
+          before_install:
+              - docker pull ${DISTRO_TYPE}
 
         # Ubuntu requires explicitly generating en_US.UTF-8 locale
-        - OS_TYPE=ubuntu
-          INSTALL_REQUIREMENTS="apt-get update; apt-get install -y gcc python3 sudo locales ed vim meson; locale-gen en_US.UTF-8"
+        - os: linux
+          env:
+            - DISTRO_TYPE=ubuntu
+              INSTALL_REQUIREMENTS="apt-get update; apt-get install -y gcc python3 sudo locales ed vim meson; locale-gen en_US.UTF-8"
+          before_install:
+              - docker pull ${DISTRO_TYPE}
 
         # Debian requires explicitly generating en_US.UTF-8 locale too
-        - OS_TYPE=debian
-          INSTALL_REQUIREMENTS="apt-get update; apt-get install -y gcc python3 python3-pip ninja-build sudo locales ed vim procps; pip3 install meson==0.44.0; sed -i '/# en_US.UTF-8 UTF-8/s/..//' /etc/locale.gen && locale-gen"
+        - os: linux
+          env:
+            - DISTRO_TYPE=debian
+              INSTALL_REQUIREMENTS="apt-get update; apt-get install -y gcc python3 python3-pip ninja-build sudo locales ed vim procps; pip3 install meson==0.44.0; sed -i '/# en_US.UTF-8 UTF-8/s/..//' /etc/locale.gen && locale-gen"
+          before_install:
+              - docker pull ${DISTRO_TYPE}
+
+        - os: osx
```

Comment:
If you add `env: - DISTRO_TYPE=macOS` then the travis dashboard would be a smidge clearer.